src/controllers/connectionManager.ts

```diff
@@ -814,6 +818,52 @@ export default class ConnectionManager {
         }
     }
 
+    public async refreshAzureAccountToken(uri: string): Promise<boolean> {
+        const profile = this.getConnectionInfo(uri);
+		if (!profile) {
+            this.vscodeWrapper.logToOutputChannel(`Connection not found for uri ${uri}`);
+			return false;
+		}
+
+        // Wait for the pending reconnction promise if any
```

Comment:
Spacing seems off in this file, make sure you format so everything is consistent